addon/components/ember-stripe-element.js

```diff
@@ -49,10 +49,12 @@ export default Ember.Component.extend({
 
   setEventListeners() {
     let stripeElement = get(this, 'stripeElement');
-    get(this, 'events').forEach(eventName => {
-      stripeElement.on(eventName, (...args) => {
-        this.sendAction(eventName, ...args);
-      });
+    stripeElement.on('blur',    () => this.sendAction('blur'));
+    stripeElement.on('focus',   () => this.sendAction('focus'));
+    stripeElement.on('change',  (...args) => {
+      let { error } = args[0];
+      set(this, 'error', error);
```

Comment:
@snewcomer I'm not sure that you do, since Elements returns a `null` or `undefined` error (can't remember which, or if both), as the `change` event updates.